isis/src/core/CMakeLists.txt

```diff
@@ -0,0 +1,113 @@
+cmake_minimum_required(VERSION 3.10)
+
+project (core VERSION 0.0.1)
```

Comment:
Is PVL really a 0.0.1 version? That would be like a just developed, alpha, probably do not use this because nothing is stable version. What are the criteria for this to be a 1.0.0 release that folks should be comfortable using?

Reply:
I think we agreed a while back that if it's going public it should be 1.0, So I imagine this should also be 1.0? We plan to do a conda deploy after this is merged.

Reply:
Agree with both, I just used 0.0.1 as a place holder

Reply:
Updated to 1.0.0